src/GovUk.Education.ExploreEducationStatistics.Common/Model/Chart/ChartDataSet.cs

```diff
@@ -11,6 +11,7 @@ public class ChartDataSet
         public List<Guid> Filters = new List<Guid>();
         public ChartDataSetLocation Location;
         public string TimePeriod;
+        public int Order;
```

Comment:
As not all charts will have this currently, we should probably set the type to `int?`, otherwise it can end up being defaulted to 0 (that's the default for `int` in C#).

Have we checked that the `order` is coming back as null/undefined from the API for charts configured with the older data sets?